docs/docs-ref-autogen/outlook/office.subject.yml

```diff
@@ -102,6 +116,19 @@ items:
 
 
       `setAsync(subject: string, callback: (result: Office.AsyncResult<void>) => void): void;`
+
+      #### Examples
+
+
+      ```javascript
+
+      Office.context.mailbox.item.subject.setAsync("New subject!", function (asyncResult) {
+          if (asyncResult.status == "failed") {

```

Comment:
As long as status cannot be weird values (numbers, []s, etc) and is just strings, this is ok but might be better as ===